Description: Remove networking tests
 Upstream is attempting connection to 192.0.2.1 on port 80, and expects
 it to always timeout. However, some Debian users may well have a web
 server running on that IP address, meaning the test would fail in such
 setup (even if this is reserved IANA IPs). Therefore, removing such bad
 tests.
 .
 Also, these tests have been failing for a reason on my arm64 builder,
 probably because it's a slow machine.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2016-07-19

--- python-eventlet-0.19.0.orig/tests/greenio_test.py
+++ python-eventlet-0.19.0/tests/greenio_test.py
@@ -74,18 +74,6 @@ class TestGreenSocket(tests.LimitedTestC
             # 3.x io write to closed file-like pbject raises ValueError
             self.assertRaises(ValueError, fd.write, b'a')
 
-    def test_connect_timeout(self):
-        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        s.settimeout(0.1)
-        gs = greenio.GreenSocket(s)
-
-        try:
-            expect_socket_timeout(gs.connect, ('192.0.2.1', 80))
-        except socket.error as e:
-            # unreachable is also a valid outcome
-            if not get_errno(e) in (errno.EHOSTUNREACH, errno.ENETUNREACH):
-                raise
-
     def test_accept_timeout(self):
         s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
         s.bind(('', 0))
@@ -95,14 +83,6 @@ class TestGreenSocket(tests.LimitedTestC
         gs = greenio.GreenSocket(s)
         expect_socket_timeout(gs.accept)
 
-    def test_connect_ex_timeout(self):
-        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-        s.settimeout(0.1)
-        gs = greenio.GreenSocket(s)
-        e = gs.connect_ex(('192.0.2.1', 80))
-        if e not in (errno.EHOSTUNREACH, errno.ENETUNREACH):
-            self.assertEqual(e, errno.EAGAIN)
-
     def test_recv_timeout(self):
         listener = greenio.GreenSocket(socket.socket())
         listener.bind(('', 0))
--- python-eventlet-0.19.0.orig/tests/socket_test.py
+++ python-eventlet-0.19.0/tests/socket_test.py
@@ -2,13 +2,6 @@ import eventlet
 from eventlet.green import socket
 
 
-def test_create_connection_error():
-    try:
-        socket.create_connection(('192.0.2.1', 80), timeout=0.1)
-    except (IOError, OSError):
-        pass
-
-
 def test_recv_type():
     # https://github.com/eventlet/eventlet/issues/245
     # socket recv returning multiple data types
